test/lib/bamboo/plug/sent_email_viewer_plug_test.exs

```diff
@@ -190,6 +190,44 @@ defmodule Bamboo.SentEmailViewerPlugTest do
     assert conn.resp_body =~ "Email not found"
   end
 
+  test "list emails over API" do
+    normalize_and_push_pair(:html_email,
+      from: "from@example.com",
+      to: ["to@example.com", {"Alice", "alice@example.com"}],
+      cc: "cc@example.com",
+      bcc: {"Bob", "bob@example.com"}
+    )
+
+    conn = conn(:get, "/api/all.json")
+
+    conn = AppRouter.call(conn, nil)
+
+    assert conn.status == 200
+    assert {"content-type", "application/json; charset=utf-8"} in conn.resp_headers
+
+    json = Bamboo.json_library().decode!(conn.resp_body)
+
+    first_email = json |> Enum.at(0)
+    assert first_email["from"] == [nil, "from@example.com"]
+    assert first_email["to"] == [[nil, "to@example.com"], ["Alice", "alice@example.com"]]
+    assert first_email["cc"] == [[nil, "cc@example.com"]]
+    assert first_email["bcc"] == [["Bob", "bob@example.com"]]
+    assert first_email["html_body"] == "<p>ohai!</p>"
+  end
+
+  test "reset emails over API" do
+    normalize_and_push_pair(:html_email)
+    conn = conn(:post, "/api/reset.json")
+
+    conn = AppRouter.call(conn, nil)
+
+    assert conn.status == 200
+    assert {"content-type", "application/json; charset=utf-8"} in conn.resp_headers
+
+    json = Bamboo.json_library().decode!(conn.resp_body)
+    assert json == %{"ok" => true}
```

Comment:
Would it make sense in here to have an assertion that the emails got cleared out?